### PR TITLE
Organize Bootstrap and non-Bootstrap SCSS variables

### LINKED ISSUE #6502: [Documentation] SCSS variables
- [x] document SCSS variables
- [x] move top level (global) variables to `variables.scss`

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #6502

### CODE REVIEW

File: source/js/buyers-guide/components/creep-vote/creep-vote.scss [hunk 1]
```diff
@@ -1,6 +1,3 @@
-$radio-button-radius: 50px;
-$btn-shadow-width: 3px;
```
Comment (mmmavis) [inline source/js/buyers-guide/components/creep-vote/creep-vote.scss:2]: old variables that are no longer in use

File: source/js/buyers-guide/components/primary-nav/primary-nav.scss [hunk 2]
```diff
@@ -1,6 +1,3 @@
-$pni-primary-nav-breakpoint: $bp-sm;
-$pni-primary-nav-container-z-index: $zindex-sticky;
```
Comment (mmmavis) [inline source/js/buyers-guide/components/primary-nav/primary-nav.scss:2]: These have been moved to [source/sass/buyers-guide/variables.scss](https://github.com/mozilla/foundation.mozilla.org/pull/6528/files#diff-e5e5421422d79fcbdb311aed60c81209c8785ca2f292dba3d6dbab4b492030f4)

File: source/sass/buyers-guide/components/header.scss [hunk 3]
```diff
@@ -1,5 +1,3 @@
-$category-nav-border-bottom-thickness: 4px;
```
Comment (mmmavis) [inline source/sass/buyers-guide/components/header.scss:1]: This variable is only used once so it doesn't really need to be a variable.

File: source/sass/buyers-guide/views/product.scss [hunk 4]
```diff
@@ -1,5 +1,3 @@
-$pni-product-breakpoint-larger: $bp-md;
```
Comment (mmmavis) [inline source/sass/buyers-guide/views/product.scss:1]: Moved to [source/sass/buyers-guide/variables.scss](https://github.com/mozilla/foundation.mozilla.org/pull/6528/files#diff-e5e5421422d79fcbdb311aed60c81209c8785ca2f292dba3d6dbab4b492030f4)

File: source/sass/glyphs.scss [hunk 5]
```diff
@@ -1,6 +1,3 @@
-$default-glyph-path: "../_images/glyphs";
```
Comment (mmmavis) [inline source/sass/glyphs.scss:1]: Moved to [source/sass/variables.scss](https://github.com/mozilla/foundation.mozilla.org/pull/6528/files#diff-d2049013185550a0f0a347ec91292d0f766e9bdd8dbdd9a94769d5fa8673723d)

File: source/sass/glyphs.scss [hunk 6]
```diff
@@ -1,6 +1,3 @@
...
-$bg-glyph-path: "../_images/buyers-guide";
```
Comment (mmmavis) [inline source/sass/glyphs.scss:2]: No longer in use.

File: source/sass/mofo-bootstrap/overrides/_variables.scss [hunk 7]
```diff
@@ -1,10 +1,11 @@
+// overrides for default Bootstrap variables
+// https://github.com/twbs/bootstrap/blob/v4.6.0/scss/_variables.scss
+
 $font-family-sans-serif: "Nunito Sans", Helvetica, Arial, sans-serif;
 $font-family-base: $font-family-sans-serif;
 
-$enable-flex: true; // Turn on Flexbox support
```
Comment (mmmavis) [inline source/sass/mofo-bootstrap/overrides/_variables.scss:4]: Old Bootstrap variable. No longer in Bootstrap v4.

File: source/sass/mofo-bootstrap/overrides/_variables.scss [hunk 8]
```diff
@@ -1,10 +1,11 @@
...
 $enable-transitions: true;
 
-$font-size-base: 1rem;
```
Comment (mmmavis) [inline source/sass/mofo-bootstrap/overrides/_variables.scss:7]: Bootstrap's default value for this variable is `1rem` as well so we shouldn't have to override it.

File: source/sass/mofo-bootstrap/overrides/_variables.scss [hunk 9]
```diff
@@ -23,7 +24,6 @@ $btn-border-radius: 0;
 $btn-border-radius-sm: 0;
 $btn-border-radius-lg: 0;
 $btn-font-weight: 700;
-$border-shadow-color: transparentize($black, 0.85);
```
Comment (mmmavis) [inline source/sass/mofo-bootstrap/overrides/_variables.scss:26]: This isn't a Bootstrap variable so I've moved to [source/sass/variables.scss](https://github.com/mozilla/foundation.mozilla.org/pull/6528/files#diff-d2049013185550a0f0a347ec91292d0f766e9bdd8dbdd9a94769d5fa8673723d)

**Review by danielfmiranda — APPROVED** [review]
Hi Mavis, 

Thanks for tagging me on this pull request, I gave it a look and everything looks good to me! 👍

I appreciate how you removed the variable that was sharing the same font size as bootstraps default, as well as adding the link to the bootstrap default page in the comments. I can see this making it easier for us to check whether or not we should have to add new variables to the codebase.

Approving!

**Comment by mmmavis** [comment]
@danielfmiranda thanks for your comment and review! :D

**Pull request merged**